staging/publishing/rules.yaml

```diff
@@ -34,6 +34,11 @@ rules:
       branch: release-1.20
       dir: staging/src/k8s.io/code-generator
     name: release-1.20
+    go: 1.15.10
```

Comment:
I wonder if that should be 1.16.1 after https://github.com/kubernetes/kubernetes/pull/100106

Reply:
no, this is for 1.20, which should use go1.15, only 1.21 and master are on 1.16

Reply:
@AlekSi that's for the master branch, release-1.20 uses go1.15.10 :)

https://github.com/kubernetes/kubernetes/blob/release-1.20/build/build-image/cross/VERSION

Reply:
Doh, missed that `-` is below. Thank you!